yarn-project/canary/src/uniswap_trade_on_l1_from_l2.test.ts

```diff
@@ -226,6 +230,8 @@ describe('uniswap_trade_on_l1_from_l2', () => {
 
   beforeAll(async () => {
     await waitForSandbox(pxe);
+    const ethCheatCodes = new EthCheatCodes(ethRpcUrl);
+    await ethCheatCodes.loadChainState(dumpedState);
```

Comment:
You are messing up the state here. You are deploying the contracts, and then you overwrite them with the state stored in `dumpedState` which don't have the rollup etc at those addresses.